spec/azure/keyvault.go

```diff
@@ -0,0 +1,6 @@
+package azure
+
+type KeyVault struct {
+	// VaultName is the Azure Key Vault name.
+	VaultName string `json:"vaultName" yaml:"vaultName"`
```

Comment:
1. Maybe just `Name`?
2. Please put some good stuff form PR description into the godoc.

Reply:
Yes I think `Name` is better. I'll change and add the extra info.

Reply:
Done